### PR TITLE
GuideConverterV2

### PR DESCRIPTION
**Pull request by jakub-pomykala**
Uploading GuideConverterV2

### CODE REVIEW

**Comment by jakub-pomykala** [comment]
DO NOT MERGE

File: .github/workflows/GuideConverterAction.yaml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    steps:
+
+      # Any prerequisite steps
+      - uses: actions/checkout@master
+
+      # Print message
+      - name: Message
+        run: |
+          echo "${{ github.event.inputs.branch }}"
+          echo "${{ github.event.inputs.guide_name }}"
```
Comment (austin0) [inline .github/workflows/GuideConverterAction.yaml:38]:
```suggestion
```

Remove as replaced by "Output message" step.

File: .github/workflows/GuideConverterAction.yaml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
```
Comment (austin0) [inline .github/workflows/GuideConverterAction.yaml:77]:
```suggestion
```

Remove as replaced by "Output message" step.

File: .github/workflows/GuideConverterAction.yaml [hunk 3]
```diff
@@ -0,0 +1,100 @@
+name: GuideConverter
+on: 
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch pushed to'     
+        required: true
+        default: 'defaulted'
```
Comment (austin0) [inline .github/workflows/GuideConverterAction.yaml:8]:
```suggestion
        default: 'qa'
```

Add a more useful default.

File: .github/workflows/GuideConverterAction.yaml [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+      guide_name:
+        description: 'Guide changed'
+        required: true
+        default: 'not given'
```
Comment (austin0) [inline .github/workflows/GuideConverterAction.yaml:12]:
```suggestion
        default: 'guide-rest-intro'
```

Add more useful default.

File: .github/workflows/GuideConverterAction.yaml [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+jobs:
+  printBranch:
+    runs-on: ubuntu-latest
+
+    steps: 
+
+      - name: Output message
+        run: |
+          echo "${{ github.event.inputs.branch }}"
+          echo "${{ github.event.inputs.guide_name }}"
```
Comment (austin0) [inline .github/workflows/GuideConverterAction.yaml:23]:
```suggestion
          echo "${{ github.event.inputs.guide_name }}"
          
```

Add line between jobs.

File: .github/workflows/GuideConverterAction.yaml [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+          git add .
+          git config --global user.email "GuidesBot@OpenLiberty.io"
+          git config --global user.name "GuidesBot"
+          git commit -m "Updated by github actions from ${{ github.event.inputs.guide_name }}"
+    
+      - uses: peter-evans/create-pull-request@v3
+        with:
+            title: "Change to ${{ github.event.inputs.guide_name }}"
+            body: "Updated by github actions, triggered by the ${{ github.event.inputs.guide_name }} repo."
+            branch: "${{ github.event.inputs.guide_name }}/actions"
```
Comment (austin0) [inline .github/workflows/GuideConverterAction.yaml:98]:
```suggestion
            branch: "${{ github.event.inputs.guide_name }}/staging"
```

Minor change to separate PR's from master/qa branches.

File: .github/workflows/GuideConverterAction.yaml [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
```
Comment (austin0) [inline .github/workflows/GuideConverterAction.yaml:59]:
```suggestion
            branch: "${{ github.event.inputs.guide_name }}/master"
```

Minor change to separate PR's from master/qa branches.

**Review by austin0 — CHANGES_REQUESTED** [review]
Few small changes, non-functional.

File: .github/workflows/GuideConverterAction.yaml (hunk 7 above)
Comment (jakub-pomykala) [reply .github/workflows/GuideConverterAction.yaml]: Done

File: .github/workflows/GuideConverterAction.yaml (hunk 6 above)
Comment (jakub-pomykala) [reply .github/workflows/GuideConverterAction.yaml]: Done

File: .github/workflows/GuideConverterAction.yaml (hunk 5 above)
Comment (jakub-pomykala) [reply .github/workflows/GuideConverterAction.yaml:23]: Done

File: .github/workflows/GuideConverterAction.yaml (hunk 4 above)
Comment (jakub-pomykala) [reply .github/workflows/GuideConverterAction.yaml]: Done

File: .github/workflows/GuideConverterAction.yaml (hunk 3 above)
Comment (jakub-pomykala) [reply .github/workflows/GuideConverterAction.yaml]: done

File: .github/workflows/GuideConverterAction.yaml (hunk 2 above)
Comment (jakub-pomykala) [reply .github/workflows/GuideConverterAction.yaml]: Done

File: .github/workflows/GuideConverterAction.yaml (hunk 1 above)
Comment (jakub-pomykala) [reply .github/workflows/GuideConverterAction.yaml]: Done

File: .github/workflows/GuideConverterAction.yaml [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+      - uses: peter-evans/create-pull-request@v3
+        with:
+            title: "Change to ${{ github.event.inputs.guide_name }}"
+            body: "Updated by github actions, triggered by the ${{ github.event.inputs.guide_name }} repo."
+            branch: "${{ github.event.inputs.guide_name }}/master"
+            base: "master" 
+        name: Create Pull Request
+
+  convertStaging:
+    name: Output message
```
Comment (austin0) [inline .github/workflows/GuideConverterAction.yaml:61]:
```suggestion
    name: Make PR to Staging
```

**Comment by jamiecoleman92** [comment]
Looks good to me. Happy for this to be merged.

**Pull request merged**